### PR TITLE
QDB-12687 - Small changes to cryptofeed

### PR DESCRIPTION
**Pull request by igorniebylski**
Removed Binance from `demo_quasar`
Changed how tables are created in `quasar` backend

### CODE REVIEW

File: cryptofeed/backends/quasar.py [hunk 1]
```diff
@@ -62,3 +73,6 @@ def format(self, data: dict):
         data['stop'] = datetime.utcfromtimestamp(data['stop'])
         data['closed'] = int(data['closed'])
         return data
+    
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange STRING, symbol SYMBOL(currency), start TIMESTAMP, stop TIMESTAMP, interval STRING, trades STRING, open DOUBLE, close DOUBLE, high DOUBLE, low DOUBLE, volume DOUBLE, closed INT64) SHARD_SIZE = {self.shard_size}'
```
Comment (solatis) [inline cryptofeed/backends/quasar.py]: Why did you call the symbol table for the column `symbol` `currency`? I also think exchange should be a symbol.

```
... (exchange SYMBOL(exchange), symbol SYMBOL(symbol), ....
```

File: cryptofeed/backends/quasar.py [hunk 2]
```diff
@@ -28,29 +34,34 @@ def _set_table_name(self, data: dict):
...
-
-    def _set_table_name(self, data: dict):
-        # setting table name depending on side
-        # trades/{exchange}/{symbol_1-symbol_2}/{side}
-        # eg. trade/coinbase/btc-usd/buy
-        super()._set_table_name(data)
-        self.table = f"{self.table}/{data['side'].lower()}"
+    
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange STRING, symbol SYMBOL(currency), side STRING, amount DOUBLE, price DOUBLE, id STRING, type STRING, receipt_timestamp TIMESTAMP) SHARD_SIZE = {self.shard_size}'
```
Comment (solatis) [inline cryptofeed/backends/quasar.py]: See feedback below

File: cryptofeed/backends/quasar.py [hunk 3]
```diff
@@ -28,29 +34,34 @@ def _set_table_name(self, data: dict):
...
-            qdbpd.write_dataframe(df, conn, conn.table(self.table), fast=True, _async=True, create=True, shard_size=self.shard_size)
+            self._create_table(conn)
+            qdbpd.write_dataframe(df, conn, conn.table(self.table), fast=True, _async=True)
 
 
 class TickerQuasar(QuasarCallback, BackendCallback):
     table_prefix = "ticker"
+    
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange STRING, symbol SYMBOL(currency), bid DOUBLE, ask DOUBLE, receipt_timestamp TIMESTAMP) SHARD_SIZE = {self.shard_size}'
```
Comment (solatis) [inline cryptofeed/backends/quasar.py]: See feedback below

**Review by solatis — CHANGES_REQUESTED** [review]

File: cryptofeed/backends/quasar.py (hunk 1 above)
Comment (igorniebylski) [reply cryptofeed/backends/quasar.py]: Right, this naming doesn't make any sense

**Comment by igorniebylski** [comment]
`exchange` and `symbol` tables are now of symbol type

**Pull request merged**